app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,181 @@
+<form class="ui form" autocomplete="off" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui yellow inverted segment">
+      <div class="ui stackable grid">
+        <div class="two wide column" style="font-size:24px;padding:20px 10px 10px 20px">
+          08:48
+        </div>
+        <div class="fourteen wide column">
+          <p>
+            Please complete registration within 10:00 minutes.
```

Comment:
The test is failing due to we're in
"After 10:00 minutes, the reservation we're holding will be released to others."